app/helpers/ui_helper.rb

```diff
@@ -5,14 +5,14 @@ def grouped_options_for_assigning_case(volunteer)
     [
       [
         "Not Assigned",
-        CasaCase
+        CasaCase.eager_load([:assigned_volunteers])
```

Comment:
did you choose `eager_load` over `includes` on purpose? `includes` will choose to use `eager_load` or `preload`. It doesn't matter that much but just curious

Reply:
In this case, I followed the recommendation of Bullet gem. 
Firstly, include was suggested, but when added a new recommendation appeared with eager_load. (I didn't take a new screenshot)